streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -124,6 +124,7 @@ public void openDB(ProcessorContext context) {
         options.setCreateIfMissing(true);
         options.setErrorIfExists(false);
         options.setInfoLogLevel(InfoLogLevel.ERROR_LEVEL);
+        options.setIncreaseParallelism(Runtime.getRuntime().availableProcessors());
```

Comment:
What is the effect of this change? I noticed there also settings like:
`options#setMaxBackgroundCompactions`
`options#setMaxBackgroundFlushes`
Do we know how/if this change effects those settings?